packages/gatsby/src/utils/webpack.config.js

```diff
@@ -579,35 +593,34 @@ module.exports = async (
   const isCssModule = module => module.type === `css/mini-extract`
 
   if (stage === `develop`) {
-    config.optimization = {
-      splitChunks: {
-        chunks: `all`,
-        cacheGroups: {
-          default: false,
-          defaultVendors: false,
-          framework: {
-            chunks: `all`,
-            name: `framework`,
-            test: FRAMEWORK_BUNDLES_REGEX,
-            priority: 40,
-            // Don't let webpack eliminate this chunk (prevents this chunk from becoming a part of the commons chunk)
-            enforce: true,
-          },
-          // Bundle all css & lazy css into one stylesheet to make sure lazy components do not break
-          // TODO make an exception for css-modules
-          styles: {
-            test(module) {
-              return isCssModule(module)
-            },
-
-            name: `commons`,
-            priority: 40,
-            enforce: true,
-          },
-        },
-      },
-      minimize: false,
-    }
+    // config.optimization = {
+    //   splitChunks: {
+    //     chunks: `all`,
+    //     cacheGroups: {
+    //       default: false,
+    //       defaultVendors: false,
+    //       framework: {
+    //         chunks: `all`,
+    //         name: `framework`,
+    //         test: FRAMEWORK_BUNDLES_REGEX,
+    //         priority: 40,
+    //         // Don't let webpack eliminate this chunk (prevents this chunk from becoming a part of the commons chunk)
+    //         enforce: true,
+    //       },
+    //       // Bundle all css & lazy css into one stylesheet to make sure lazy components do not break
+    //       // TODO make an exception for css-modules
+    //       styles: {
+    //         test(module) {
+    //           return isCssModule(module)
+    //         },
+    //         name: `commons`,
+    //         priority: 40,
+    //         enforce: true,
+    //       },
+    //     },
```

Comment:
`splitChunks` need to be rethinkied. No function callbacks allowed here (`test`, `name`, etc)